lib/core/plugin/pluginsManager.js

```diff
@@ -625,13 +625,20 @@ class PluginsManager {
     }
   }
 
-  _initApi (plugin) {
+  async _initApi (plugin) {
     for (const [controller, definition] of Object.entries(plugin.instance.api)) {
       debug(
         '[%s][%s] starting api controller registration',
         plugin.name,
         controller);
 
+      if (await this.kuzzle.ask('kuzzle:api:funnel:controller:isNative', controller)) {
```

Comment:
IMHO for this kind of usage it could be interesting if `kuzzle.ask` does not always returns a promise

Reply:
true, but it is also interesting (more, IMHO) that the return value of `kuzzle.ask` is always the same type.